xilem_core/src/view.rs

```diff
@@ -305,7 +305,7 @@ where
         ctx: &mut Context,
         element: Mut<'_, Self::Element>,
     ) {
-        if core::mem::take(&mut view_state.dirty) || !Rc::ptr_eq(self, prev) {
+        if core::mem::take(&mut view_state.dirty) || !Self::ptr_eq(self, prev) {
```

Comment:
This is probably a downgrade in code quality :(

Reply:
But feel free.